PluginClasses/ZPTVACellViewController.swift

```diff
@@ -102,8 +107,64 @@ class ZPTVACellViewController : CACellViewController {
         }
     }
     
+    private func isFavoriteSupported() -> UIButton?{
+        guard let layout = self.currentComponentModel()?.style["layout_name"] as? String,
+            let favoriteBtn  = favoritesButton,
+            layout == "Family_5_horizontal_list_13" || layout == "Family_5_grid_11" || layout == "Family_5_grid_12"  else{
```

Comment:
Please filter to check all values in an array https://stackoverflow.com/questions/30828074/check-if-array-contains-part-of-a-string-in-swift

Reply:
done